services-js/registry-certs/pages/_document.tsx

```diff
@@ -47,14 +41,8 @@ export default class extends Document {
 
   constructor(props: Props) {
```

Comment:
Can just delete this `constructor` now.

Reply:
Hrm, my editor gives me a ts error without it. Will figure out what I’m missing.

Reply:
yeah actually ok to leave it for now. Will be able to remove it when Next TypeScript types work right.